### ISSUE TITLE
[5-dev] Site changes due to upgrade to Dart 2 SDK + new tooling

### ISSUE THREAD

**Issue by chalin** [dart-lang/site-webdev#1366]
Below is a list of pages that will need to be updated due to changes in the Dart 2 SDK and associated tooling. (This is independent of updates necessary because of upgrades to Angular 5.)

- Angular
  - deployment.md: e.g., mentions `dart_to_js_script_rewriter`, `pub build` etc.
- Tools
  - Drop Dartium, (add Dart Code?)
  - Subpages: dart2js, ddc, pub build & serve
- Articles
  - low-level-html/embedding-in-html.md: e.g., mentions `browser` package
- FAQ

**Comment by kwalrath** [comment]
Thanks for starting this list!

**Comment by kwalrath** [comment]
Of course, we'll need to update anything that mentions core API that has changed (e.g. due to constants being changed from SCREAMING_CAPS to lowerCamelCase). I'm hoping that our tests will catch most or all of these.
